File: 58.length-of-last-word/58.length-of-last-word.js
/*
 * @lc app=leetcode id=58 lang=javascript
 *
 * [58] Length of Last Word
 */
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    if(!s) return 0
    
    return s.trim().split(' ').slice(-1)[0].length
};

File: 58.length-of-last-word/58.length-of-last-word.go
/*
 * @lc app=leetcode id=58 lang=golang
 *
 * [58] Length of Last Word
 */
import (
	"strings"
)

func lengthOfLastWord(s string) int {
    if len(s) == 0 {
			return 0
		}

		ss := strings.Trim(s, " ")
		s1 := strings.Fields(ss)
		s3 := ""
		
		for key, value := range s1 {
			if len(s1) == (key + 1) {
				s3 = value
			}
		}

		return len(s3)
}

